.github/workflows/test-asan.yml

```diff
@@ -30,9 +30,9 @@ jobs:
     if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     env:
-      CC: clang
-      CXX: clang++
-      LINK: clang++
+      CC: clang-12
+      CXX: clang++-12
+      LINK: clang++-12
```

Comment:
Is `LINK` here neccessary ?

Reply:
I don't remember why `LINK` was necessary (to force `clang` linker instead of `ld` when linking? idk that's the only thing I can think of). We can always remove and see what happens

Reply:
I do think it was more of a GitHub Actions thing and less of a gnu thing

Reply:
Ok I remember now. `ld` would consume too much memory for ASAN to be possible on Actions, given the size of the instances available (see https://github.com/nodejs/node/pull/32776). I'm almost sure `LINK` is not standard but is used by gyp. What I'm not entirely sure is if `CXX` being set to `clang++-12` is enough. Again, someone would need to experiment (specifically looking at peak memory usage when `LINK` is set or not).